bigtable/src/create_dev_instance.php

```diff
@@ -25,81 +25,81 @@
 // Include Google Cloud dependencies using Composer
 require_once __DIR__ . '/../vendor/autoload.php';
 
-if (count($argv) < 3 || count($argv) > 5) {
-    return printf("Usage: php %s PROJECT_ID INSTANCE_ID CLUSTER_ID [LOCATION_ID]" . PHP_EOL, __FILE__);
-}
-list($_, $projectId, $instanceId, $clusterId) = $argv;
-$locationId = isset($argv[4]) ? $argv[4] : 'us-east1-b';
-
 // [START bigtable_create_dev_instance]
-
 use Google\Cloud\Bigtable\Admin\V2\BigtableInstanceAdminClient;
 use Google\Cloud\Bigtable\Admin\V2\Instance;
 use Google\Cloud\Bigtable\Admin\V2\Cluster;
 use Google\Cloud\Bigtable\Admin\V2\StorageType;
 use Google\Cloud\Bigtable\Admin\V2\Instance\Type as InstanceType;
 use Google\ApiCore\ApiException;
 
-/** Uncomment and populate these variables in your code */
-// $projectId = 'The Google project ID';
-// $instanceId = 'The Bigtable instance ID';
-// $clusterId = 'The Bigtable cluster ID';
-// $locationId = 'The Bigtable region ID';
-
-
-$instanceAdminClient = new BigtableInstanceAdminClient();
-
-$projectName = $instanceAdminClient->projectName($projectId);
-$instanceName = $instanceAdminClient->instanceName($projectId, $instanceId);
-
-
-printf("Creating a DEVELOPMENT Instance" . PHP_EOL);
-// Set options to create an Instance
-
-$storageType = StorageType::HDD;
-$development = InstanceType::DEVELOPMENT;
-$labels = ['dev-label' => 'dev-label'];
-
-
-# Create instance with given options
-$instance = new Instance();
-$instance->setDisplayName($instanceId);
-$instance->setLabels($labels);
-$instance->setType($development);
-
-// Create cluster with given options
-$cluster = new Cluster();
-$cluster->setDefaultStorageType($storageType);
-$cluster->setLocation(
-    $instanceAdminClient->locationName(
-        $projectId,
-        $locationId
-    )
-);
-$clusters = [
-    $clusterId => $cluster
-];
-// Create development instance with given options
-try {
-    $instanceAdminClient->getInstance($instanceName);
-    printf("Instance %s already exists." . PHP_EOL, $instanceId);
-} catch (ApiException $e) {
-    if ($e->getStatus() === 'NOT_FOUND') {
-        printf("Creating a development Instance: %s" . PHP_EOL, $instanceId);
-        $operationResponse = $instanceAdminClient->createInstance(
-            $projectName,
-            $instanceId,
-            $instance,
-            $clusters
-        );
-        $operationResponse->pollUntilComplete();
-        if (!$operationResponse->operationSucceeded()) {
-            print('Error: ' . $operationResponse->getError()->getMessage());
+/**
+ * Create a development Bigtable instance
+ * @param string $projectId The Google Cloud project ID
+ * @param string $instanceId The ID of the Bigtable instance to be generated
+ * @param string $clusterId The ID of the cluster to be generated
+ * @param string $locationId The Bigtable region ID where you want your instance to reside
+ */
+function create_dev_instance($projectId, $instanceId, $clusterId, $locationId = 'us-east1-b')
+{
```

Comment:
Same as the other PR, let's add typehints for everything!
```suggestion
function create_dev_instance(
    string $projectId,
    string $instanceId,
    string $clusterId,
    string $locationId = 'us-east1-b'
): void {
```